pylint/checkers/base.py

```diff
@@ -1525,15 +1525,26 @@ def _name_became_keyword_in_version(name, rules):
 
 class DocStringChecker(_BasicChecker):
     msgs = {
-        'C0111': ('Missing %s docstring', # W0131
-                  'missing-docstring',
-                  'Used when a module, function, class or method has no docstring.'
-                  'Some special methods like __init__ doesn\'t necessary require a '
-                  'docstring.'),
         'C0112': ('Empty %s docstring', # W0132
                   'empty-docstring',
                   'Used when a module, function, class or method has an empty '
                   'docstring (it would be too easy ;).'),
+        'C0114': ('Missing module docstring',
+                  'missing-module-docstring',
+                  'Used when a module has no docstring.'
+                  'Empty modules doesn\'t necessary require a docstring.',
+                  {'old_names': [('C0111', 'missing-docstring')]}),
+        'C0115': ('Missing class docstring',
+                  'missing-class-docstring',
+                  'Used when a class has no docstring.'
+                  'Even an empty class must have a docstrings.',
+                  {'old_names': [('C0111', 'missing-docstring')]}),
+        'C0116': ('Missing function or method docstring',
+                  'missing-function-docstring',
+                  'Used when a function or method has no docstring.'
+                  'Some special methods like __init__ doesn\'t necessary '
+                  'require a docstring.',
```

Comment:
I know that this message existed already, but now that we're changing the message id can we fix the message as well? Perhaps: "Some special methods like __init__ do not require a docstring"